### ISSUE TITLE
Sync option between any Arabic Recitation & Translation Audio

### ISSUE THREAD

**Issue by Abdullah076** [quran/quran_android#908]
Option to sync between any Arabic recitation & translation audios in other languages, so that any verse can be heard with its recitation & translation (Available in ayat app)

For example:
If the sync is turned on;
If Surah fatiha 1:1 is played, then translation (Ibrahim Walk English/ or other Audio translations) will be played automatically after Arabic recitation. Then arabic of 1:2 will be played, and again translation will be played after that.
Basically mixing the Arabic recitation & Audio translations.
if sync is turned off; then only selected audio will play.
